Makefile

```diff
@@ -12,6 +12,7 @@ lint:
 test: all
 	QIIMETEST= pytest
 
+# install pytest-xdist plugin for the `-n auto` argument.
```

Comment:
https://github.com/qiime2/q2cli/blob/master/ci/recipe/meta.yaml#L31

Reply:
oh thanks, I dunno why I didn't think to look there first